app/routes/ping.js

```diff
@@ -0,0 +1,9 @@
+const PingRoute = function(app, pkg){
+
+  app.get("/", (req, res) => {
```

Comment:
@donnapep I thought about that and I think we should add a routing for the files such as `localhost:9494/files/?url=`.  What do you think? cheers